purchase_order_partner_ref_mandatory/readme/DESCRIPTION.rst

```diff
@@ -1 +1 @@
-Adds Partner Ref Mandatory in purchase order.
+Add Partner Ref field in purchase order
```

Comment:
I think this is a wrong description, because partner reference field already exists. With this addon we simply make this field more important in some views.